libraw/src/main/java/com/homesoft/photo/libraw/LibRaw.java

```diff
@@ -36,6 +39,34 @@ public class LibRaw implements AutoCloseable {
 
     long mNativeContext;
 
+    public static class CameraWhiteBalance {
+        public final float temp;
+        public final float r;
+        public final float g;
+        public final float b;
+        public final float g1;
+        public CameraWhiteBalance(float temp, float r, float g, float b, float g1) {
+            this.temp = temp;
+            this.r = r;
+            this.g = g;
```

Comment:
Just curious on your "g" notation.  Libraw has these as G1 and G2, but I couldn't find any authoritative sources.  I was looking for Bayer Patterns, but I might looking in the wrong spot.